sdx_controller/messaging/rpc_queue_consumer.py

```diff
@@ -3,25 +3,33 @@
 import logging
 import os
 import threading
+import time
 from queue import Queue
 
 import pika
 
 from sdx_controller.handlers.lc_message_handler import LcMessageHandler
 from sdx_controller.utils.parse_helper import ParseHelper
 
-MQ_HOST = os.environ.get("MQ_HOST")
 # subscribe to the corresponding queue
 SUB_QUEUE = os.environ.get("SUB_QUEUE")
-
+MQ_SRVC = os.environ.get("MQ_SRVC")
```

Comment:
It looks like `MQ_HOST` was renamed to `MQ_SRVC`, but not with good reason. Everywhere else (environment files, tests, CI setup) we use `MQ_HOST`.

Reply:
MQ_SRVC has the Docker service name value and MQ_HOST has the IP address. I suspect the test error is caused because adding authentication is not tested, and MQ_HOST is not used anymore.

Reply:
Please continue to use `MQ_HOST`, and please make just the necessary changes to make password authentication work.

Reply:
No problem! So, password authentication is currently working fine. I'll go ahead and bring back MQ_HOST, and if the test still doesn't go as planned, I'm happy to keep working on it. But I'd really appreciate your help in figuring out what's causing the issue and finding a solution to make the deployment happen. Thanks so much!

Reply:
I'm looking into the test failures.

Reply:
I think the problem is that not all environment variables are set when running the tests: especially `MQ_SRVC` (which should be `MQ_HOST`) in this case.  It will also fail if `MQ_USER` or `MQ_PASS` is not set.

It would be nice to have some checks to see that all configuration items are set, but that is a separate issue (see #27) from this.  We should definitely use some defaults in this PR -- like using RabbitMQ default username/password of `guest`/`guest` when no username and password environment variable is set, as I've stated in another review comment.

In the meanwhile, could you set up a local development environment and try running the tests yourself?

Reply:
You probably want to reset your PR branch, and cherry-pick the relevant commits to the one relevant file, like so:

```
$ git switch luisdev
$ git fetch origin
$ git reset --hard origin/main
$ git cherry-pick 9ba4b92 c0d3459
```

Now, rename `MQ_SRVC` back to `MQ_HOST`, and use user/auth defaults.  Also remove the `time.sleep()` calls. We should not add unnecessary latency, unless there is a good reason.  And then update this PR with:

```
$ git push <remote> luisdev --force-with-lease
```

Obviously replace `<remote>` with whatever remote name that you are using.